src/com/google/javascript/jscomp/CommandLineRunner.java

```diff
@@ -790,6 +791,9 @@
     )
     private String packageJsonEntryNames = null;
 
+    @Option(name = "--error_format", usage = "Specifies format for error messages.")
```

Comment:
Minor but can the possible values be added in the usage?
Something like: `Specifies format for error messages. Options: STANDARD (default), JSON`

Reply:
I believe args4j adds them automatically for us.

Reply:
ok, I didn't know that. I've added this comment because other options in this file (compilation_level for example) list all the possible values.

Reply:
It only works for ENUM types. Compilation Level has some very non-standard parsing logic around it which requires us to manually list the values.